gallery/src/DarkModeModal/DarkModeModal.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useEffect, useState } from 'react';
+
+import {
+  NxButton,
+  NxModal,
+  NxForm,
+  NxFormGroup,
+  NxToggle,
+  useToggle,
+  NxFieldset,
+  NxRadio,
+  NxTooltip,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+
+const DarkModeModal = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [enableModeChange, toggleEnableModeChange] = useToggle(false);
+  const [mode, setMode] = useState<string | null>(null);
+
+  const { classList } = document.documentElement;
```

Comment:
It would be cleaner with this line moved into the `useEffect` as well

Reply:
> It would be cleaner with this line moved into the `useEffect` as well

moved into the useEffect here: [803b3a0](https://github.com/sonatype/sonatype-react-shared-components/pull/782/commits/803b3a0a0d10d335b14f55ba3ce34acb4c51a022)